x/lend/types/keys.go

```diff
@@ -12,6 +12,10 @@ const (
 	ModuleAcc1 = "cmdx"
 	ModuleAcc2 = "atom"
 	ModuleAcc3 = "osmo"
+	ModuleAcc4 = "axlusdc"
+	ModuleAcc5 = "statom"
+	ModuleAcc6 = "evmos"
+	ModuleAcc7 = "gusdc"
```

Comment:
will keep adding this module account manually for every asset we add or do we have something for this?

Reply:
Rn, we are adding manually. We can have something, so we don't need to add it manually in the future.

Reply:
So if the chain needs to add token XYZ in lend module in some time it needs a software upgrade for this, am I  correct on this ?

Reply:
Yes